src/test/java/jag/ParserTest.java

```diff
@@ -0,0 +1,80 @@
+package jag;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class ParserTest {
+    @Test
+    public void parse_list_test() throws AExceptions {
+        Command actual = Parser.parse("list");
+        assertTrue(actual instanceof ListCommand, "Expected an instance of ListCommand");
+    }
+
+    @Test
+    public void parse_mark_test() throws AExceptions {
+        Command actual = Parser.parse("mark ");
+        assertTrue(actual instanceof MarkCommand, "Expected an instance of ListCommand");
+    }
+
+    @Test
+    public void parse_unmark_test() throws AExceptions {
+        Command actual = Parser.parse("unmark ");
+        assertTrue(actual instanceof MarkCommand, "Expected an instance of ListCommand");
+    }
+
+    @Test
+    public void parse_todods_test() throws AExceptions {
```

Comment:
nit: perhaps using this three part naming convention format mentioned in the java conventions for naming your test case methods could make them more descriptive?

3 part format: `featureUnderTest_testScenario_expectedBehavior()`